sites/page-details/index.json-ld

```diff
@@ -82,5 +83,346 @@
+					{
+						"@type": "source-code",
+						"description": "Détails de la page par défaut :",
+						"code": {
+							"@type": [ "@id", "rdf:HTML" ],
+							"@value": "samples/page-details-v3-fr.html#default"
+						}
+					},
+					{
+						"@type": "source-code",
+						"description": "Détails de la page avec Outil de rétroaction sur la page :",
+						"code": {
+							"@type": [ "@id", "rdf:HTML" ],
+							"@value": "samples/page-details-v3-fr.html#pft"
+						}
+					},
+					{
+						"@type": "source-code",
+						"description": "Détails de la page avec Partagez cette page :",
+						"code": {
+							"@type": [ "@id", "rdf:HTML" ],
+							"@value": "samples/page-details-v3-fr.html#share"
+						}
+					}
+					,
+					{
+						"@type": "source-code",
+						"description": "Détails de la page avec Outil de rétroaction sur la page et Partagez cette page :",
+						"code": {
+							"@type": [ "@id", "rdf:HTML" ],
+							"@value": "samples/page-details-v3-fr.html#pftShare"
+						}
+					}
+				]
+			}
+		},
+		{
+			"@id": "_:implement_pd_gcweb",
+			"iteration": "_:iteration_pd_3",
+			"name": {
+				"en": "GCWeb Jekyll",
+				"fr": "GCWeb Jekyll"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers trying to implement this component within a GCWeb Jekyll site.",
+				"fr": "Cette implémentation est destinée aux développeurs essayant d'implémenter ce composant dans un site GCWeb Jekyll."
+			},
+			"instructions": {
+				"en": [
+					"To show the \"Page feedback tool\", in the page's front-matter, set the property <code>feedback</code> to <code>true</code>",
+					"To show the \"Share this page\" button, in the page's front-matter, set the property <code>share</code> to <code>true</code>"
+				],
+				"fr": [
+					"Pour afficher l'\"Outil de rétroaction sur la page\", dans l'en-tête de la page, définissez la propriété <code>feedback</code> à <code>true</code>.",
+					"Pour afficher le bouton \"Partagez cette page\", dans l'en-tête de la page, définissez la propriété <code>share</code> à <code>true</code>."
+				]
+			},
+			"sample": {
+				"en": [
+					{
+						"@type": "source-code",
+						"description": "Code sample:",
+						"code": "\"feedback\": true, \n\"share\": true"
+					}
+				],
+				"fr": [
+					{
+						"@type": "source-code",
+						"description": "Exemple de code:",
+						"code": "\"feedback\": true, \n\"share\": true"
+					}
+				]
+			}
+		}
+	],
+	"iteration": [
+		{
+			"@id": "_:iteration_pd_3",
+			"name": "Page details - Iteration 3",
+			"date": "2023-03",
+			"detectableBy": "section.pagedetails",
+			"predecessor": "_:iteration_pd_2",
+			"fixes": [
+				"Changed <footer> to <section> as <footer> element does not provide a semantic break."
+			],
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-v3-en.html#full"
```

Comment:
The selector `#full` don't exist in the sample file `samples/page-details-v3-en.html`